README.md

```diff
@@ -13,3 +13,19 @@ Girl Develop It is a nonprofit organization that exists to provide affordable an
 ## Documentation
 
 We have migrated our documentation over to Read The Docs! Learn more about how to contribute to our code base by following along here: [http://gdi-new-site.readthedocs.org/](http://gdi-new-site.readthedocs.org/).
+
+###How to Vagrant
+_Note: This will move to the official documentation once the Vagrant dev environment is adopted into the staging branch._
+
+To  set up the vagrant environment locally:
+1. [Install VirtualBox](https://www.virtualbox.org/)
+2. [install Vagrant](https://www.vagrantup.com/downloads.html)
+3. [clone the git repository](https://github.com/girldevelopit/gdi-website.git)
+4. $ vagrant up
```

Comment:
We will just want to add that this is a big setup process so to give at least 5 min to see things pull through...